modules/utils/docker/entrypoint_nats.sh

```diff
@@ -33,10 +33,31 @@ else
       sleep 1
   done
 
+  CBMA=$(extract_features_value "CBMA" $YAML_FILE)
+  if [ "$CBMA" == "false" ]; then
```

Comment:
Probably not important here since hopefully there won't be a need to disable CBMA, but for other such tests in the future: it might be worth making the test case insensitive, otherwise someone might write FEATURE=False or FEATURE=FALSE or maybe even FEATURE=0 and not realize that these do not disable the feature.

Reply:
CBMA setting check will be removed once CBMA is default config.